cmd/inspect/inspect_policy.go

```diff
@@ -126,6 +148,7 @@ func inspectPolicyCmd() *cobra.Command {
 		},
 	}
 
+	cmd.Flags().StringVarP(&policyRef, "policy", "p", "", "reference to the policy configuration, either EnterpriseContractPolicy Kubernetes custom resource reference [<namespace>/]<name>, or inline JSON or YAML of the `spec` part")
 	cmd.Flags().StringArrayVarP(&sourceUrls, "source", "s", []string{}, "policy source url. multiple values are allowed")
```

Comment:
Should it say, "Likely to be ignored when using the --policy flag"?

Reply:
It's not, one can use `--policy` and `--source` if they so desire. Thought it would be useful in "what if I add this source" use case.